.github/workflows/OCV-Nightly-RISCV.yaml

```diff
@@ -345,3 +346,228 @@ jobs:
         $TEST_RUNNER \
           OPENCV_TEST_DATA_PATH=${REMOTE_DATA} \
             ${REMOTE_BIN}/opencv_test_video ${TEST_OPT}
+
+  BuildAndTestNDS:
+    env:
+      TEST_RUNNER: '/opt/andes/bin/qemu-riscv64 -cpu andes-ax25 -L /opt/andes/sysroot'
+      TEST_OPT: ''
+      BRANCH: '${{ matrix.branch }}'
+    strategy:
+      max-parallel: 1
+      matrix:
+        branch: ['4.x', '5.x']
+    runs-on: opencv-ru-lin-riscv
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: quay.io/opencv-ci/opencv-ubuntu-22.04-riscv-nds:20240709
+      volumes:
+        - /mnt/cache/git_cache:/home/ci/git_cache
+        - /mnt/cache/ci_cache/opencv:/home/ci/.ccache
+        - /mnt/cache/binaries_cache:/home/ci/binaries_cache
+        - /home/build/.ssh:/root/.ssh
+    steps:
+    - name: Define proper HOME path
+      timeout-minutes: 60
+      run: echo "HOME=/home/ci" >> $GITHUB_ENV
+    - name: Fetch opencv
+      timeout-minutes: 60
+      run: |
+        rm -rf ${{ env.SRC_OPENCV }}
+        git clone \
+          --single-branch \
+          --branch ${{ env.BRANCH }} \
+          --reference-if-able ${{ env.GIT_CACHE_DOCKER }}/opencv.git \
+          https://github.com/opencv/opencv.git \
+          ${{ env.SRC_OPENCV }}
+    - name: Fetch opencv_contrib
+      timeout-minutes: 60
+      run: |
+        rm -rf ${{ env.SRC_OPENCV_CONTRIB }}
+        git clone \
+          --single-branch \
+          --branch ${{ env.BRANCH }} \
+          --reference-if-able ${{ env.GIT_CACHE_DOCKER }}/opencv_contrib.git \
+          https://github.com/opencv/opencv_contrib.git \
+          ${{ env.SRC_OPENCV_CONTRIB }}
+    - name: Fetch opencv_extra
+      timeout-minutes: 60
+      run: |
+        rm -rf ${{ env.SRC_OPENCV_EXTRA }}
+        git clone \
+          --single-branch \
+          --branch ${{ env.BRANCH }} \
+          --reference-if-able ${{ env.GIT_CACHE_DOCKER }}/opencv_extra.git \
+          https://github.com/opencv/opencv_extra.git \
+          ${{ env.SRC_OPENCV_EXTRA }}
+    - name: Configure OpenCV
+      timeout-minutes: 60
+      run: |
+        cmake -G Ninja \
+          -S ${{ env.SRC_OPENCV }} \
+          -B ${{ env.BUILD_DIR }} \
+          -DCMAKE_TOOLCHAIN_FILE=$HOME/opencv/platforms/linux/riscv64-andes-gcc.toolchain.cmake \
+          -DRISCV_GCC_INSTALL_ROOT=/opt/andes \
+          -DCPU_BASELINE=RVV \
+          -DCPU_BASELINE_REQUIRE=RVV \
+          -DRISCV_RVV_SCALABLE=ON \
```

Comment:
Do we really want to enable and require RVV on the Andes platform? I'm not sure if the andes compiler/emulator supports RVV

Reply:
No, it's a mistake.